jsapp/js/account/usage/usageProjectBreakdown.tsx

```diff
@@ -1,11 +1,198 @@
+        const uid = assetParts[assetParts.length - 2];
+        return {
+          ...projectResult,
+          uid: uid,
+        };
+      });
+
+      setProjectData((prevData) => {return {...prevData, results: updatedResults};});
+    }
+  };
+
+ const onFetchAssetsDataDone = (assetsData: any, updatedResults: ProjectResult[]) => {
+  if (isComponentMounted) {
+    const updatedData = updatedResults
+      .filter((projectResult) => {
+        const matchingAsset = assetsData.results.find(
+          (asset: ProjectViewAsset) => asset.uid === projectResult.uid
+        );
+
+        // Include the asset in the updatedData only if it is a survery
+        return matchingAsset?.asset_type === 'survey';
+      })
+      .map((projectResult) => {
+        const matchingAsset = assetsData.results.find(
+          (asset: ProjectViewAsset) => asset.uid === projectResult.uid
+        );
+
+        if (matchingAsset) {
+          return {
+            ...projectResult,
+            assets: [matchingAsset],
+            deployment_status: matchingAsset.deployment_status,
+          };
+        }
+        return projectResult;
+      });
+
+    setProjectData((prevData: Project) => {
+      return {
+        ...prevData,
+        results: updatedData as ProjectResult[],
+      };
+    });
+  }
+};
+
+  // This code will be updated to only have one api call when the asset_usage api is updated
+  // Right now, you may need to refresh or re-navigate to the page multiple times as the Loading Spinner can be shown indefinitely
+  useEffect(() => {
+    const usageUrl = `${ROOT_URL}/api/v2/asset_usage`;
+
+    const fetchData = $.ajax({
+      dataType: 'json',
+      method: 'GET',
+      url: usageUrl,
+    })
+      .done(onFetchAssetsDone)
+      .then(() => {
+        const assetsUrl = `${ROOT_URL}/api/v2/assets`;
+
+        return $.ajax({
+          dataType: 'json',
+          method: 'GET',
+          url: assetsUrl,
+        })
+          .done((assetsData) =>
+            onFetchAssetsDataDone(assetsData, projectData.results)
+          )
+          .fail(onAnyFail);
+      })
+      .fail(onAnyFail);
+
+    return () => {
+      setIsComponentMounted(false);
+    };
+
+    function onAnyFail(error: any) {
+      console.error('Error fetching data:', error);
+    }
+  }, [projectData.results]);
+
+  if (projectData.results.length === 0 || projectData.results.some((result) => !result.assets)) {
+    return <LoadingSpinner/>;
+  }
+
+  return (
+    <div className={styles.root}>
+      <table>
+        <thead>
+          <tr>
+            {/* Note: the projectData count will be 0 for now as the api needs to be updated */}
+            <th className={styles.projects}>{t('##count## Projects').replace('##count##', projectData.count)}</th>
+            <th className={styles.wrap}>{t('Submissions (Total)')}</th>
+            <th className={styles.wrap}>{t('Submissions (This billing period)')}</th>
```

Comment:
You can get the billing period (`"month"`/`"year"`) to display here using `getSubscriptionInterval()` from `stripe.api.ts` 👍 (Probably in a `useEffect()` hook)

Reply:
As was pointed out, the design calls for `this billing period`, not `this {BILLING_PERIOD}` - resolving this.